docs/source/qvm.rst

```diff
@@ -20,7 +20,7 @@ Within pyQuil, there is a :py:class:`~pyquil.api.QVM` object and a :py:class:`~p
 the exposed APIs of the QVM and QPU servers, respectively.
 
 On this page, we'll learn a bit about the :ref:`QVM <qvm_use>` and :ref:`QPU <qpu>`. Then we will
-show you how to use them from pyQuil with a :ref:`quantum_computer`.
+show you how to use them from pyQuil with :ref:`quantum_computer`.
```

Comment:
The `quantum_computer` ref expands to "The QuantumComputer", which makes "a The QuantumComputer" awkward.

Reply:
Yuck. If I'm understanding correctly this now reads "... from pyQuil with The QuantumComputer" which is just as ugly 😠 (not that that is your fault appleby).

Reply:
Yeah, still gross. I considered changing the title of the referenced section to be just "QuantumComputer", but the preceding sections are titled "The Quantum Virtual Machine" and "The Quantum Processing Unit", so I left it alone for consistency.

My rst-fu is rusty, but I will see if I can figure out how to link to this section while just providing my own link text.

Reply:
Done. Text now reads "... from pyQuil with a QuantumComputer".

Reply:
Ignorable nitpick: I would prefer "... from pyQuil with a QuantumComputer object."

Reply:
Done.